### ISSUE TITLE
format changes for tickets

### ISSUE THREAD

**Issue by PundaChen** [Minehut/Meta#781]
**Platform**
website

**Describe the feature**
change the support ticket format depending on the choice and add report & appeal options with this

**How the feature is useful**
So the user only provides information that is actually necessary for the ticket

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by tarna** [comment]
I would also suggest changing the format for the other ticket categories, not just for reports and appeals. Because I don't think all those Minecraft related questions aren't needed for tickets related to the website or purchasing credits.

**Comment by PundaChen** [comment]
> change the support ticket format depending on the choice

**Comment by tarna** [comment]
> > change the support ticket format depending on the choice

yep I read the issue wrong thinking you were only suggesting it for the reports and appeals 😔
